packages/react-components/src/organisms/__tests__/SharedResearchCard.test.tsx

```diff
@@ -64,7 +64,7 @@ it('displays link component when link property present', () => {
   const { getByText } = render(
     <SharedResearchCard {...libraryCardProps} link={'https://example.com'} />,
   );
-  const link = getByText(/external\slink/i).closest('a');
+  const link = getByText(/external\slink/i, { selector: 'div' }).closest('a');
```

Comment:
```suggestion
  const link = getByText(/external\slink/i, { selector: 'a *' }).closest('a');
```
I think, if I understood correctly what's going on